src/System.Runtime.WindowsRuntime/src/project.json

```diff
@@ -8,6 +8,12 @@
       "imports": [
         "dotnet5.4"
       ]
+    },
+    "uap10.1": {
```

Comment:
Also delete `src\System.Runtime.WindowsRuntime\src\netcore50aot\project.json` and remove references to it from the `.csproj` file.